go/oasis-test-runner/cmd/root.go

```diff
@@ -47,13 +56,30 @@ var (
 		Run:   runList,
 	}
 
-	rootFlags = flag.NewFlagSet("", flag.ContinueOnError)
+	rootFlags       = flag.NewFlagSet("", flag.ContinueOnError)
+	TestParamsFlags = flag.NewFlagSet("", flag.ContinueOnError)
+	TestParamsMask  = "params.%s.%s"
 
 	cfgFile string
+	numRuns int
 
-	scenarioMap      = make(map[string]scenario.Scenario)
+	Scenarios        = make(map[string]scenario.Scenario)
 	defaultScenarios []scenario.Scenario
-	scenarios        []scenario.Scenario
+
+	// oasis-test-runner-specific metrics.
+	upGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "up",
```

Comment:
Probably best to avoid using `"up"` as it's already used generated automatically by prometheus: https://prometheus.io/docs/concepts/jobs_instances/#automatically-generated-labels-and-time-series 

Should at least prefix with similarly as we do for other metrics.

Reply:
Yeah, "oasis_up" seems a better idea. Also for cleaning up any invalid benchmark results.